Makefile

```diff
@@ -48,7 +48,11 @@ config:
 	@echo "// Automatically generated by \`make config BUILDDIR=$(BUILDDIR)\`, do not edit." >> llvm_config_$(GOOS).go
 	@echo "" >> llvm_config_$(GOOS).go
 	@echo "// #cgo CPPFLAGS: $(shell $(CONFIG) --cppflags)" >> llvm_config_$(GOOS).go
-	@echo "// #cgo CXXFLAGS: -std=c++11" >> llvm_config_$(GOOS).go
+	@if [ $(VERSION_MAJOR) -gt 9 ]; then \
+	echo "// #cgo CXXFLAGS: -std=c++14" >> llvm_config_$(GOOS).go; \
+	else \
+	echo "// #cgo CXXFLAGS: -std=c++11" >> llvm_config_$(GOOS).go; \
+	fi
```

Comment:
I tested with `-std=c++14` for LLVM 9 and it seems to work just fine, so this condition should not be necessary.

Reply:
Yea, I doubt that it would break LLVM 9, but I left it there in case old systems didn't support it (but could still build LLVM with c++11).